code/modules/research/techweb/all_nodes.dm

```diff
@@ -432,7 +432,7 @@
 	display_name = "Advanced Cybernetic Implants"
 	description = "Upgraded and more powerful cybernetic implants."
 	prereq_ids = list("neural_programming", "cyber_implants","integrated_HUDs")
-	design_ids = list("ci-toolset", "ci-surgery", "ci-reviver")
+	design_ids = list("ci-toolset", "ci-surgery", "ci-reviver", "ci-nutrimentplus")

```

Comment:
nutrimentplus is shit

Reply:
whats the point of having x and better x on the same node? same for cutter

Reply:
agreed